<launch>
    <node name="ugv_control_node" pkg="sunray_ugv_control" type="ugv_control_node" output="screen">
        <param name="ugv_id" value="1" />
        <param name="flag_printf" value="true" />
        <param name="pose_source" value="2" />
        <param name="ugv_control_param/Kp_xy" value="1.4" />
        <param name="ugv_control_param/Kp_yaw" value="0.9" />
        <param name="ugv_control_param/max_vel_xy" value="0.5" />
        <param name="ugv_control_param/max_vel_yaw" value="50.0 / 180.0 * M_PI" />
        <param name="ugv_geo_fence/max_x" value="10.0" />
        <param name="ugv_geo_fence/min_x" value="-10.0" />
        <param name="ugv_geo_fence/max_y" value="10.0" />
        <param name="ugv_geo_fence/min_y" value="-10.0" />
        <param name="resolution" value="0.6" />
        <param name="inflate" value="0.2" />

        <!-- map generator 模块使用 -->
        <!-- odom_type 1:从 odom_topic中获取 其他:从ugv_state中获取-->
        <param name="odom_type" value="2" />
        <param name="odom_topic" value="/car_odom" />
        <param name="laser_topic" value="/scan" />
        <param name="octomap_topic" value="/octomap_full" />
        <param name="occupancy_topic" value="/occupancy_grid" />
    </node>
</launch>
